src/replication.c

```diff
@@ -2563,13 +2564,138 @@ int sendCurrentOffsetToReplica(client *replica) {
     return C_OK;
 }
 
+static int dualChannelRepliacationHandleHandshake(connection *conn, char **err) {
+    serverLog(LL_DEBUG, "Received first reply from primary using rdb connection.");
+    /* AUTH with the primary if required. */
+    if (server.primary_auth) {
+        char *args[] = {"AUTH", NULL, NULL};
+        size_t lens[] = {4, 0, 0};
+        int argc = 1;
+        if (server.primary_user) {
+            args[argc] = server.primary_user;
+            lens[argc] = strlen(server.primary_user);
+            argc++;
+        }
+        args[argc] = server.primary_auth;
+        lens[argc] = sdslen(server.primary_auth);
+        argc++;
+        *err = sendCommandArgv(conn, argc, args, lens);
+        if (*err) {
+            serverLog(LL_WARNING, "Sending command to primary in dual channel replication handshake: %s", *err);
```

Comment:
I think it is very valuable to have `dual channel` captured in the log statement as well but it looks like we are not doing it in a very consistent way today. How about starting a new pattern? Then the operator can easily filter in/out dual-chan log entries (and other log entries in the future)
```suggestion
            serverLog(LL_WARNING, "[Dual Channel] Sending command to primary in replication handshake: %s", *err);
```

Reply:
I agree, I'll fix it for all dual channel logs

Reply:
Let's not make a local decision about a new pattern. The pattern, if anything, would be `<Dual channel>` as is used by the modules.

Reply:
I'm reopening this action item for further discussion.
https://github.com/valkey-io/valkey/issues/986